frontend/src/hocs/withHostCommunication/types.ts

```diff
@@ -121,6 +122,9 @@ export type IHostToGuestMessage = {
   | {
       type: "CLEAR_CACHE"
     }
+  | {
+      type: "SEND_APP_HEARTBEAT"
```

Comment:
new window message type. Sent by SiS parent window to iframe.

should it just be `SEND_HEARTBEAT`?